### PR TITLE
#213 enable offence dz out of udr

### PR DESCRIPTION
**Pull request by PavelFilo**
- version upgrade,
- enable no zone while offence type DZ
- add turf library to know if location is in polygon and display error based on this information

### CODE REVIEW

File: app.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
     name: 'Skenovacia aplikácia',
     slug: 'enforcement-scan-app',
     scheme: 'enforcement-scan-app',
-    version: '1.0.2',
```
Comment (radoslavzeman) [inline app.config.js:6]: If it's bugfix, we bump last number, if it's new feature or a change (not a bugfix), we bump middle number.

Don't be afraid to bump middle number :)

File: app/(app)/offence/index.tsx [hunk 2]
```diff
@@ -79,13 +94,17 @@ const OffencePage = () => {
             <Field label={t('offence.vehicle')}>
               <TextInput
                 value={ecv}
-                className="font-source-500medium"
+                autoCapitalize="characters"
+                className="font-belfast-700bold text-[18px] color-black"
                 isDisabled={!!role?.actions.scanCheck}
                 onChangeText={(value) => setOffenceState({ ecv: value })}
```
Comment (radoslavzeman) [inline app/(app)/offence/index.tsx:100]: Shouldn't we use the same formatting function as in MPA?
trim, toUpperCase, replace whitespaces?

File: app/(app)/offence/index.tsx [hunk 3]
```diff
@@ -79,13 +94,17 @@ const OffencePage = () => {
...
               />
             </Field>
 
-            <Field label={t('offence.location')}>
+            <Field
+              label={t('offence.location')}
+              errorMessage={isTouched && isLocationError ? t('offence.outOfZone') : undefined}
```
Comment (radoslavzeman) [inline app/(app)/offence/index.tsx:106]: What is touched? :D

File: components/map/location-map/LocationMapScreen.tsx [hunk 4]
```diff
@@ -36,16 +39,19 @@ const LocationMapScreen = ({ role }: Props) => {
 
   const [centerCoordinate, setCenterCoordinate] = useState(location)
 
+  const isDZ = offenceType === OffenceTypeEnum.Dz
```
Comment (radoslavzeman) [inline components/map/location-map/LocationMapScreen.tsx:42]: `isDz`

File: translations/sk.json [hunk 5]
```diff
@@ -21,6 +21,7 @@
   "offence.offenceResolutionPlaceholder": "Vyberte druh vyriešenia",
   "offence.offenceType": "Druh priestupku",
   "offence.offenceTypePlaceholder": "Vyberte druh priestupku",
+  "offence.outOfZone": "Ste mimo zony, vyberte zónu",
```
Comment (radoslavzeman) [inline translations/sk.json:24]: "zóny"

**Review by radoslavzeman — APPROVED** [review]
Nice 💪

File: app/(app)/offence/index.tsx (hunk 3 above)
Comment (PavelFilo) [reply app/(app)/offence/index.tsx:106]: touched means that the form was submitted

**Pull request merged**